cups/translate-attrs.go

```diff
@@ -67,6 +67,16 @@ func getUUID(printerTags map[string][]string) string {
 }
 
 func getState(printerTags map[string][]string) cdd.CloudDeviceStateType {
+	// offline-report reason is set when printer is offline (e.g. USB cable is disconnected)
```

Comment:
Ok. Probably this could work if you call `getVendorState` and check for `offline`.